setup.py

```diff
@@ -72,7 +72,7 @@ def find_meta(meta):
 ]
 INSTALL_REQUIRES = [
     "attrs",
-    "click",
```

Comment:
Thank you so much for this! 

It looks like it was added in 7.1 (see changelog [here](https://click.palletsprojects.com/en/8.0.x/options/#choice-options)) - would you mind updating it real quick?

Reply:
Sure thing; done!